src/components/OperationRow.js

```diff
@@ -1,36 +1,45 @@
 /* @flow */
 import React, { PureComponent } from "react";
 import { View, StyleSheet } from "react-native";
+import { connect } from "react-redux";
 import { RectButton } from "react-native-gesture-handler";
 import { Trans } from "react-i18next";
 import { getOperationAmountNumber } from "@ledgerhq/live-common/lib/operation";
 
 import type { Account, Operation } from "@ledgerhq/live-common/lib/types";
 
+import { createStructuredSelector } from "reselect";
 import LText from "./LText";
 import CurrencyUnitValue from "./CurrencyUnitValue";
 import CounterValue from "./CounterValue";
 import CurrencyIcon from "./CurrencyIcon";
-import OperationIcon from "./OperationIcon";
 
+import OperationIcon from "./OperationIcon";
 import colors from "../colors";
 import OperationRowDate from "./OperationRowDate";
 import LiveLogo from "../icons/LiveLogoIcon";
 import Spinning from "./Spinning";
+import { currencySettingsForAccountSelector } from "../reducers/settings";
+import type { CurrencySettings } from "../reducers/settings";
 
 type Props = {
   operation: Operation,
   account: Account,
   navigation: *,
   multipleAccounts?: boolean,
   isLast: boolean,
+  currencySettings: CurrencySettings,
 };
 
 const placeholderProps = {
   width: 40,
   containerHeight: 20,
 };
 
+const mapStateToProps = createStructuredSelector({
+  currencySettings: currencySettingsForAccountSelector,
+});
```

Comment:
I'm a bit worried about overall performance of this. like we did on desktop, I would really prefer we don't connect the full row but that we make a dedicated `OperationConfirmationIcon` component that is connected (that would be an intermediary light component). that way, we avoid unecessary redraw each time the redux updates (which is after each sync for instance)

Reply:
👍 The current version should address that, minimizing the redraw